app/controllers/admin/fields_controller.rb

```diff
@@ -114,7 +114,22 @@ def subform
   protected
 
   def field_params
-    params[:field].permit!
+    params.require(:field).permit(
```

Comment:
I *doubt* that anyone would have extended the field class while using it as an engine; but we should make sure we write a migration guide for anywhere we went for explicit param whitelisting, suggesting people will want to .permit() their customised attributes